.github/workflows/tests.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
   run-tests:
     strategy:
       matrix:
-        os: [macos-11, macos-12, windows-2019, windows-2022]
+        os: [macos-12, windows-2019, windows-2022]
```

Comment:
Is macos-11 EOL/unsupported?

Reply:
Nvm, just saw the PR message

Reply:
Yes, they removed the runners in June. (I accidentally didn't save the description edit before you got to it :-) )https://github.blog/changelog/2024-05-20-actions-upcoming-changes-to-github-hosted-macos-runners/#macos-11-deprecation-and-removal